backend/src/Database.hs

```diff
@@ -162,6 +162,12 @@ updatePlayerName pid name = lift $ do
     set player [PlayerName =. val (normalizeName name), PlayerDisplayName =. val name]
     where_ (player ^. PlayerId ==. val pid)
 
+deleteGameReport :: (MonadIO m, MonadLogger m) => Key GameReport -> DBAction m ()
+deleteGameReport rid = lift $ do
+  delete $ do
+    report <- from $ table @GameReport
+    where_ (report ^. GameReportId ==. val rid)
```

Comment:
Is this a no-op if no such `rid` is found, or will it throw an error?

Reply:
Currently a no-op. Should we respond with an error? That seems like a good idea

Reply:
No-op seems fine, could just have the response include a 1 or 0 for number of reports deleted in that case? The frontend could interpret a 0 as a failure 👌